pkg/storage/secret/metadata/data/secure_value_set_version_to_active.sql

```diff
@@ -0,0 +1,8 @@
+UPDATE
+  {{ .Ident "secret_secure_value" }}
+SET
+  {{ .Ident "active" }} = ({{ .Ident "version" }} = {{ .Arg .Version}})
```

Comment:
Any particular reason why we set `active` to `true` in an expression like this rather than explicitly doing?:
```suggestion
  {{ .Ident "active" }} = true,
  {{ .Ident "version" }} = {{ .Arg .Version }}
```

Reply:
We potentially updating more than one row.

Given these rows:

[{ns=default, name=b, active=true, version=1}, 
 {ns=default, name=a, active=true, version=1},
 {ns=default, name=a, active=false, version=2}]

the query will change the db to this:

[{ns=default, name=b, active=true, version=1}, 
 {ns=default, name=a, active=false, version=1}, <-- changed
 {ns=default, name=a, active=true, version=2}] <-- changed